src/main/java/com/eternalcode/parcellockers/user/UserRepositoryJdbcImpl.java

```diff
@@ -36,10 +38,14 @@ public CompletableFuture<Void> save(User user) {
     public CompletableFuture<Optional<User>> findByName(String name) {
         return CompletableFuture.supplyAsync(() -> {
             try (ResultSet resultSet = this.jdbcConnectionProvider.executeQuery("SELECT * FROM `users` WHERE `name` = ? LIMIT 1".replace("?", name))) {
```

Comment:
Nie no to samo co wyżej, #replace tutaj nie jest bindowaniem parametrów do zapytania